test/support/TaxYearUtils.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package utils
+package support
```

Comment:
What's the difference between `support` and `utils`? Is the intention to eventually migrate everything under `utils` to `support`?

Reply:
Ooh got it!